external-crates/move/crates/move-analyzer/editors/code/README.md

```diff
@@ -9,7 +9,7 @@ language [documentation](https://docs.sui.io/concepts/sui-move-concepts).
 2. Open the command palette (`⇧` + `⌘` + `P` on macOS, or use the menu item *View > Command Palette...*) and
    type **Extensions: Install Extensions**. This will open a panel named *Extensions* in the
    sidebar of your Visual Studio Code window.
-3. In the search bar labeled *Search Extensions in Marketplace*, type **Mysten**. The Move extension 
+3. In the search bar labeled *Search Extensions in Marketplace*, type **Mysten**. The Move extension
```

Comment:
Could we use the full extension name, `mysten.move`?

Also, I feel like typically, the documented path is Command-P `ext install mysten.move`.

Reply:
`mysten.move` is a unique identifier for this extension but it's not really featured on the extension's landing page, which I think would make it confusing if we used it in the docs

![image](https://github.com/user-attachments/assets/bda6d4bc-deae-4afa-a0f8-6ee5978ae0e3)

> Also, I feel like typically, the documented path is Command-P ext install mysten.move

Tried this but it did not work for me... The procedure we list (inherited from the old extension) is pretty foolproof I'd think and only have to be done once (unless you heard complaints from developers that this did not work for them in which case we should certainly adjust it)